docs/projects/even-odd/_posts/2018-12-31-python.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Even-Odd in Python
+layout: default
+last-modified: 2018-12-31
+tags: ['python', 'even-odd', 'syntax']
+authors: "Chase Stevens"
+---
+
+# Even_Odd
+
+Oftentimes when dealing with things and numbers, we want to split up what we're working with into two groups. This property, known as parity, is useful to know. With Python, we can create a function to check the parity of any given integer.
```

Comment:
Maybe I'm just unfamiliar with "parity" in the context you're trying to provide, but this paragraph confuses me a little.